scripts/isolate_tests.py

```diff
@@ -96,7 +74,7 @@ def write_cases(f, tests):
 
 def extract_and_write(f, path):
     if docs:
-        cases = extract_docs_cases(path)
+        cases = extract_docs_cases(path, ".. code-block:: solidity")
```

Comment:
I think that with this change we'll now be silently skipping all the `::` and `.. code::` blocks. Maybe we should report them as errors/warnings instead?

Reply:
Well, there are some legimate cases for that.
What I am more wondering is if we should have all those checks/validations in the same script...

Reply:
Well, Sphinx can't check that for us automatically so I guess this is the next best place. If it fails, you'll see the error in CI at least.